evergreen/evergreen.yml

```diff
@@ -357,6 +470,354 @@ tasks:
       commands:
         - func: publish-snapshot
 
+    - name: test-ocsp-rsa-valid-cert-server-staples-ca-responder
```

Comment:
See https://github.com/mongodb/specifications/tree/master/source/ocsp-support/tests#integration-tests-permutations-to-be-tested

Reply:
Note: .NET on Windows currently doesn't support stapling, and neither does the server for now. Thus, all stapled OCSP tests will behave like nonstapled OCSP tests, as noted under "Note" in the section linked above.

Reply:
Ah true. I think leading with a verb is a bit easier to understand and don't mind keeping it the way it is TBH :)

Reply:
maybe, but it will create some inconsitensy.. Maybe not imporatant, but still inconsistency